src/app/components/create-button/create-button.component.ts

```diff
@@ -0,0 +1,14 @@
+import { Component, Output, EventEmitter } from '@angular/core';
+
+@Component({
+  selector: 'app-create-button',
+  templateUrl: './create-button.component.html',
+  styleUrls: ['./create-button.component.css'],
+})
+export class CreateButtonComponent {
+  @Output() createButtonClick = new EventEmitter();
+
+  onClick() {
+    this.createButtonClick.emit();
```

Comment:
We could use `clickEvent` as the output name, then this expression will be `clickEvent.emit()` which sound more expressive IMO

Reply:
done